pkgs/applications/audio/ttaenc/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchurl }: stdenv.mkDerivation (finalAttrs: {
```

```suggestion
{ lib
, stdenv
, fetchurl
}:

stdenv.mkDerivation (finalAttrs: {
```

small formatting thing but it makes this a lot nicer to read